README.md

```diff
@@ -175,4 +178,3 @@ license.
 ## Authors
 
 Written by [Dark](https://darklang.com). We may be [hiring](https://darklang.com/careers).
-
```

Comment:
Apologies to the changes for this file, I suspect this was VSCode being helpful and doing some auto-formatting once I saved.

Shall I revert the other changes (they don't appear to have any affect on how the file looks when rendered)

Reply:
Nah, it's fine. A nice format is good.